bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/builder/BoundaryEventBuilderTest.java

```diff
@@ -108,4 +108,20 @@ void testErrorBoundaryEventErrorCodeVariableAndErrorMessageVariableCanAllBeSet()
         .extracting(ZeebeError::getErrorCodeVariable, ZeebeError::getErrorMessageVariable)
         .containsExactly(tuple("errorCode", "errorMessage"));
   }
+
+  @Test
+  void testErrorBoundaryEventDoesNotContainExtensionElements() {
```

Comment:
🔧 Let's clarify what we're trying to verify here: `DoesNotContainZeebeErrorExtensionElement`

We can also add a description to the assertion below:
```java
assertThat(errorEventDefinition.getUniqueChildElementByType(ExtensionElements.class))
  .describedAs("Expect that ZeebeError extension element is not added to error event definition")
  .isNull();
```

If we would ever add other extension elements to error event definitions, then it would be easier to understand why this test would fail now, and what it was trying to achieve.